test/bats/base-test.bats

```diff
@@ -6,6 +6,41 @@ BATS_TESTS_DIR=${BATS_TESTS_DIR:-test/bats/tests}
 WAIT_TIME=60
 SLEEP_TIME=1
 
+@test "base test without cert rotator" {
+    teardown() {
+        echo "cleaning up"
+        wait_for_process ${WAIT_TIME} ${SLEEP_TIME} 'kubectl delete pod demo --namespace default --force --ignore-not-found=true'
+        wait_for_process ${WAIT_TIME} ${SLEEP_TIME} 'kubectl delete pod demo-oci-image --namespace default --force --ignore-not-found=true'
+        wait_for_process ${WAIT_TIME} ${SLEEP_TIME} 'kubectl delete pod demo1 --namespace default --force --ignore-not-found=true'
+    }
+    run kubectl apply -f ./library/default/template.yaml
+    assert_success
+    sleep 5
+    run kubectl apply -f ./library/default/samples/constraint.yaml
+    assert_success
+    sleep 5
+    # validate certificate store status property shows success
+    run bash -c "kubectl get certificatestores.config.ratify.deislabs.io/ratify-notary-inline-cert -n gatekeeper-system -o yaml | grep 'issuccess: true'"
+    assert_success
+    run kubectl run demo --namespace default --image=registry:5000/notation:signed
+    assert_success
+    # notation signature with OCI Artifact manifest format
+    if [[ $IS_OCI_1_1 == "true" ]]; then
+        run kubectl run demo-oci-image --namespace default --image=registry:5000/notation:ociartifact
+        assert_success
+    fi
+    run kubectl run demo1 --namespace default --image=registry:5000/notation:unsigned
+    assert_failure
+}
+
+@test "cert rotator test" {
+    helm uninstall ratify --namespace gatekeeper-system
```

Comment:
I tried to use `helm upgrade` which works without providing `--set featureFlags.RATIFY_CERT_ROTATION=true` flag. But it always failed to deploy Ratify pod with it. I checked logs which shows the pod is already up but helm regards it as not ready. What's more interesting is that the same `helm upgrade` command works on my local machine.

Reply:
oh that's interesting. Maybe there's another resource other than the pod that is failing to be upgraded due some cpu/mem limitation? Either way I'm personally ok with your solution.